docarray/typing/url/any_url.py

```diff
@@ -35,6 +38,31 @@ def _to_node_protobuf(self) -> 'NodeProto':
 
         return NodeProto(text=str(self), type=self._proto_type_name)
 
+    @classmethod
+    def validate(
+        cls: Type[T],
+        value: Union[T, np.ndarray, Any],
+        field: 'ModelField',
+        config: 'BaseConfig',
+    ) -> T:
+        import os
+
+        try:
+            if not value.startswith('http') and not os.path.isabs(value):  # type: ignore
```

Comment:
why is mypy complaining here?

Reply:
Ok makes sense. So seems like the TypeError is something that we expect during normal/correct execution of the program? That is usually a bit of a code smell, can comparatively slow etc. Can we proactively check the types to avoid this?